energyplus_regressions/tests/diffs/tbl_resources/eplustbl_empty_table_mod.htm

```diff
@@ -0,0 +1,92 @@
+  2018-11-20
+  17:26:37
+ - EnergyPlus</title>
+</head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<body>
+<p><a href="#toc" style="float: right">Table of Contents</a></p>
+<a name=top></a>
+<p>Program Version:<b>EnergyPlus, Version 9.0.1-a7c9cc14ce, YMD=2018.11.20 17:26</b></p>
+<p>Tabular Output Report in Format: <b>HTML</b></p>
+<p>Building: <b>Bldg</b></p>
+<p>Environment: <b>DENVER CENTENNIAL ANN CLG 1% CONDNS DB=>MWB ** </b></p>
+<p>Simulation Timestamp: <b>2018-11-20
+  17:26:37</b></p>
+<hr>
+<p><a href="#toc" style="float: right">Table of Contents</a></p>
+<a name=AnnualBuildingUtilityPerformanceSummary::EntireFacility></a>
+<p>Report:<b> Annual Building Utility Performance Summary</b></p>
+<p>For:<b> Entire Facility</b></p>
+<p>Timestamp: <b>2018-11-20
+    17:26:37</b></p>
+<b>Values gathered over         0.00 hours</b><br><br>
+<b>WARNING: THE REPORT DOES NOT REPRESENT A FULL ANNUAL SIMULATION.</b><br><br>
+<b></b><br><br>
+<b>Site and Source Energy</b><br><br>
+<!-- FullName:Annual Building Utility Performance Summary_Entire Facility_Site and Source Energy-->
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">Total Energy [GJ]</td>
+    <td align="right">Energy Per Total Building Area [MJ/m2]</td>
+    <td align="right">Energy Per Conditioned Building Area [MJ/m2]</td>
+  </tr>
+  <tr>
+    <td align="right">Total Site Energy</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+  </tr>
+  <tr>
+    <td align="right">Net Site Energy</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+  </tr>
+  <tr>
+    <td align="right">Total Source Energy</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+  </tr>
+  <tr>
+    <td align="right">Net Source Energy</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+  </tr>
+</table>
+<br><br>
+<b>Site to Source Energy Conversion Factors</b><br><br>
+<!-- FullName:Annual Building Utility Performance Summary_Entire Facility_Site to Source Energy Conversion Factors-->
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">Site=>Source Conversion Factor</td>
+  </tr>
+  <tr>
+    <td align="right">Electricity</td>
+    <td align="right">       3.167</td>
+  </tr>
+  <tr>
+    <td align="right">Natural Gas</td>
+    <td align="right">       1.084</td>
+  </tr>
+</table>
+<br><br>
+<b>Building Area</b><br><br>
+<!-- FullName:Annual Building Utility Performance Summary_Entire Facility_Building Area-->
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">Area [m2]</td>
+  </tr>
+  <tr>
+    <td align="right">None</td>
+    <td align="right"></td>
```

Comment:
New test mod file, I've cleared out the building area table here to represent a size change.